### ISSUE TITLE
Add warning about token length being greater than GPT limits

### ISSUE THREAD

**Issue by hargup** [mpoon/gpt-repository-loader#26]
I have an internal library for counting tokens on files and directories, should probably opensource it.

**Comment by tddschn** [comment]
Is it based on OpenAI's `tiktoken` library? @hargup

**Comment by hargup** [comment]
Yes it is based on `tiktoken`, just open sourced it. https://github.com/felvin-search/token-count
